experiments/ClimaEarth/user_io/postprocessing.jl

```diff
@@ -0,0 +1,100 @@
+## helpers for user-specified IO
+include("debug_plots.jl")
+include("diagnostics_plots.jl")
+
+"""
+    postprocess_sim(sim_mode::AbstractSlabplanetSimulationMode, cs, postprocessing_vars)
+
+If they exist, perform conservation checks, for any slabplanet simulation, and then call
+`common_postprocessing` to perform common postprocessing tasks that are common to all simulation types.
```

Comment:
update docstring since `common_postprocessing` is called at the beginning of the function now